third_party/istio-head/istio-ci-mesh.yaml

```diff
@@ -24,6 +24,8 @@ spec:
       autoscaleMax: 10
       cpu:
         targetAverageUtilization: 60
+      env:
+        PILOT_ENABLED_SERVICE_APIS: true
```

Comment:
this is default in latest istio fwiw

Reply:
Oh, thank you! Istio v1.10+ enabled it by default.Then, we should update to 1.10 by https://github.com/knative-sandbox/net-istio/issues/631 first then add this option in 1.9 only.